srcpkgs/walld/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'walld'
+pkgname=walld
+version=1.4.1
+revision=1
+archs="i686 x86_64"
```

Comment:
this will probably build on all arches, let's drop the restriction for now

Reply:
It should on all glibc arches. So I will set it as ~*-musl

Reply:
Why does it not build on musl? That should be fixed with a patch and sent upstream.

Reply:
It reassigned stdin, stdout, and stderr. There is a patch that just closes them. It will be included in upstream 1.4.2.